components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,40 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				gap: 12px;
+				height: calc(var(--d2l-table-cell-height) + var(--d2l-sortable-button-height));
 				letter-spacing: inherit;
 				margin: 0;
-				padding: 0;
+				padding-inline-start: var(--d2l-sortable-button-padding);
 				text-decoration: none;
+				text-wrap: nowrap;
+				width: 100%;
 			}
 			button::-moz-focus-inner {
 				border: 0;
 			}
 			button:disabled {
 				opacity: 0.5;
 			}
-			button:hover {
-				text-decoration: underline;
-			}
 			button:focus-visible,
 			button:${unsafeCSS(getFocusPseudoClass())} {
-				border-radius: 0.2rem;
+				border-radius: var(--d2l-sortable-button-border-radius);
 				box-shadow: 0 0 0 2px #ffffff, 0 0 0 4px var(--d2l-color-celestine);
+				height: var(--d2l-sortable-button-focus-height);
+				margin-inline-start: calc(var(--d2l-sortable-button-padding) * var(--d2l-sortable-button-focus-margin));
 				outline-style: none;
+				padding-inline-start: calc(var(--d2l-sortable-button-padding) * (1 - var(--d2l-sortable-button-focus-margin)));
+				width: var(--d2l-sortable-button-focus-width);
```

Comment:
Does this need to use a `box-shadow`? I think a lot of it might be able to be simplified with a `border` instead?

Reply:
With a border instead it looks like this:

![Screenshot (119)](https://github.com/BrightspaceUI/core/assets/24593380/e30cf966-96b5-46c0-b198-8cb8a16a38e9)

I think that might be why the original code used box-shadow

Reply:
If all the new properties are removed and the box-shadow gets replaced with a border then the sizing does look correct but there is a slight flickering in the position of the text, so that's likely why a box-shadow was used instead. I'm still thinking it would be possible to simplify this a bit though.

Reply:
Ok, I was able to find a way to use border instead